.github/workflows/build_rocks.yaml

```diff
@@ -16,7 +16,9 @@ on:
         default: "ghcr.io"
       runs-on:
         type: string
-        description: Image runner for building the images
+        description: |
+          Image runner for building the images or a sequence of tags for self-hosted runners
+          (e.g. "[self-hosted, linux]")
         default: ubuntu-22.04
```

Comment:
Does the default need to change?  Will `"{{ fromJSON("ubuntu-22.04") }}"` still evaluate correctly?